src/popover.coffee

```diff
@@ -234,8 +234,15 @@ Ember.Widgets.PopoverComponent.reopenClass
   rootElement: '.ember-application'
   hideAll: -> $(document).trigger('popover:hide')
 
-  popup: (options) ->
-    @hideAll()
+  ###*
+   * Shows a popup with options used for PopoverComponent.
+   * @param {Object} options All options used for PopoverComponent.
+   * @param {boolean} [hideOthers=true] Indicates if other popover should be hidden when a new one
+   *    is shown. By default, it's set to true.
+  ###
+  popup: (options, hideOthers = true) ->
+    if hideOthers
+      @hideAll()
     rootElement = options.rootElement or @rootElement
```

Comment:
This is unrelated to my change. I don't think having default values if options is missing is a no-brainer - there are advantages to having options effectively required (enforced at runtime, although it could be improved with an explicit error message). This change should be considered independently from this PR